last-call.xml

```diff
@@ -25,7 +25,7 @@ layout: null
       <item>
         <title>{{ eip.title | xml_escape }}</title>
         <description>{{ description | xml_escape }}</description>
-        <pubDate>{{ eip.date | date: "%a, %d %b %Y %H:%M:%S %z" }}</pubDate>
+        <pubDate>{{ eip.created | date_to_rfc822 }}</pubDate>
```

Comment:
Previously this pointed to a non-existent `date` field. Also none of the dates were RFC-822 compliant.